content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,155 @@ aliases:
+Run the following:
+
+{{< keep-url >}}
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save this output so it can be used later.
+
+Continuous Queries are replaced by Tasks in InfluxDB 2.0
+To convert your CQs to Flux, see the [Tasks documentation](/influxdb/v2.0/process-data/get-started/).
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+you may want to research using Telegraf instead.
+InfluxDB 2.0 doesn't support these protocols directly.
+Use Telegraf to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+<!-- TODO edit this section -->
+
+There is no [subscriptions API](/influxdb/v1.8/administration/subscription-management/) in InfluxDB 2.0.
+To continue using Kapacitor, update the Kapacitor configuration to use the appropriate security credentials per [1.x compatibility API docs 1](/influxdb/v2.0/reference/api/influxdb-1x/)
+(using the appropriate user and token to allow access).
+
+<!-- from TH:-->
+
+OSS users can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scenarios:
+
+1. Kapacitor Batch-style TICKscripts work with the 1.x read compatible API
+   Existing Kapacitor user credentials should continue to work using the [1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+2. Kapacitor Stream-style TICKscripts will not work using the subscription API (as there isn’t one present in 2.0).
+   We recommend writing data directly to *both* Kapacitor and InfluxDB.
+   This allows stream tasks to continue to work.
+   (See dual writes using Telegraf.)
+   If you use other mechanisms to feed data in, Telegraf could be used as an interstiatial layer to feed both Kapacitor and InfluxDB.
+   There are other architectures and mechanisms possible.
+
+### User migrations
+
+`influxd upgrade` attempts to migrate existing 1.x users and their permissions.
+However, *administrative users are not migrated*.
+
+If using an admin user for vizualition or Chronograf administrative functions, you may want to create a new read only user before upgrading.
+Admin rights are granted to the primary user created in the 2.0 setup process via the `influxd upgrade` command.
+(This provides you with the opportunity to re-assess who should be granted admin level access as a part of your InfluxDB 2.0 setup.)
+
+### Dashboards
+
+You can ontinue to use your existing dashboard/visualization tools with InfluxDB 2.0 via the [1.x read compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+Mappings are created as part of the upgrade process to ensure existing users can execute queries with the appropriate permissions.
+Reconfigure your data sources to point to your new InfluxDB 2.0 OSS instance with the appropriate security credentials.
```

Comment:
not sure why this is here? the key bit is outlined below.

Reply:
Which sentence(s) seem extraneous?

Reply:
line 95...it doesn't make any sense given what follows.  Because the point of "in-place" is that if you've got a non-admin cred, it should just work.  But, we are telling folks to change it....they only need to change it, if they had an admin cred.